docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.7"
 
 services:
   refractr:
-    image: ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com/refractr:${REFRACTR_VERSION}
+    image: refractr:${REFRACTR_VERSION}
```

Comment:
I'm unclear how this resolves to ECR and not Dockerhub.

Reply:
https://github.com/mozilla-it/refractr/pull/46/files#diff-5ddda38bd048b0606552d6c1890f9259L269-R254

Reply:
`refractr` locally, tagged with ECR url only for publish.  Makes your local docker images not so damn spammy with the gigantic ECR url.